src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -599,7 +599,8 @@ class MavlinkStreamSysStatus : public MavlinkStream
 			int lowest_battery_index = 0;
 
 			for (int i = 0; i < ORB_MULTI_MAX_INSTANCES; i++) {
-				if (battery_status[i].connected && (battery_status[i].remaining < battery_status[lowest_battery_index].remaining)) {
+				if (battery_status[i].connected && ((!battery_status[lowest_battery_index].connected)
```

Comment:
Could you add a comment explaining this logic?

Reply:
OK,
![image](https://user-images.githubusercontent.com/30532769/90364280-ba16e400-e096-11ea-8072-bdcc00f9d20d.png)
![image](https://user-images.githubusercontent.com/30532769/90364316-c1d68880-e096-11ea-8d33-7e24afb32780.png)